capstone/src/components/User/Adventurer/Preference/Preference.jsx

```diff
@@ -1,143 +1,56 @@
 import {
-  Box, HStack, FormControl, FormLabel, Switch, Button, useToast,
```

Comment:
I know it's a bit tricky, but one way to split up this pull request is to move refactoring into its own PR.